plugins/CoreHome/javascripts/noreferrer.js

```diff
@@ -0,0 +1,18 @@
+/*!
+ * Piwik - free/libre analytics platform
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+$(function () {
+    if ($.browser.msie && parseInt($.browser.version) === 10) {
+        $(document).on('click', 'a[rel~="noreferrer"]', function (event) {
+            event.preventDefault();
+            var a = event.currentTarget;
+            var w = window.open(a.href, a.target || '_self');
+            if (/\bnoopener\b/.test(a.rel)) {
+                w.opener = null;
```

Comment:
maybe we should always set w.opener = null ?

Reply:
If we want IE 10 to behave like other browsers, we should only do it when `noopener` is specified. Otherwise IE 10 will be more strict/secure that other browsers, and this may cause compatibility issues.

Or, in other words, if we want the `noopener` functionality on all links, we should add it to all links instead of just the IE 10 shim.